### PR TITLE
Couple details

### PR DESCRIPTION
**Pull request by ricab**
Just a couple of minor fixes.

### CODE REVIEW

File: src/client/cli/cmd/networks.h [hunk 1]
```diff
@@ -15,8 +15,8 @@
  *
  */
 
-#ifndef MULTIPASS_LIST_COMMANDS_H
-#define MULTIPASS_LIST_COMMANDS_H
+#ifndef MULTIPASS_NETWORKS_H
+#define MULTIPASS_NETWORKS_H
```
Comment (luis4a0) [inline src/client/cli/cmd/networks.h:19]: Oops, my fault. I forgot these guards when renaming the commands. Thanks @ricab for fixing this :)

File: src/client/cli/cmd/CMakeLists.txt [hunk 2]
```diff
@@ -24,8 +23,8 @@ add_library(commands STATIC
   info.cpp
   launch.cpp
   list.cpp
-  networks.cpp
   mount.cpp
+  networks.cpp
```
Comment (luis4a0) [inline src/client/cli/cmd/CMakeLists.txt:27]: Yeah, also I forgot the alphabetic order, thanks.

**Review by luis4a0 — APPROVED** [review]
Yeah!

**Comment by luis4a0** [comment]
bors delegate

**Comment by ricab** [comment]
Alright, thanks!

bors r+

**Comment by ricab** [comment]
bors r=luis4a0

**Pull request merged**